### ISSUE TITLE
ayatana-indicator-datetime crash no clock visiable on panel 

### ISSUE THREAD

**Issue by franksmcb** [AyatanaIndicators/ayatana-indicator-datetime#16]
Bug reported in Launchpad. Forwarding here per request of @sunweaver 
https://bugs.launchpad.net/ubuntu/+source/ayatana-indicator-datetime/+bug/1893244

Ubuntu MATE 20.10 updated today to new ayatana-indicators-datetime

Expected:
Clock shows in panel

Actual:
No clock in panel.
ayatana-indicator-datetime crashes on login and creates .crash file

ProblemType: Bug
DistroRelease: Ubuntu 20.10
Package: ayatana-indicator-datetime 0.8.0-1
ProcVersionSignature: Ubuntu 5.8.0-18.19-generic 5.8.4
Uname: Linux 5.8.0-18-generic x86_64
ApportVersion: 2.20.11-0ubuntu45
Architecture: amd64
CasperMD5CheckResult: skip
CurrentDesktop: MATE
Date: Thu Aug 27 10:56:38 2020
InstallationDate: Installed on 2019-12-13 (258 days ago)
InstallationMedia: Ubuntu-MATE 20.04 LTS "Focal Fossa" - Alpha amd64 (20191213)
SourcePackage: ayatana-indicator-datetime
UpgradeStatus: Upgraded to groovy on 2020-06-02 (85 days ago)

Crash file included.

[_usr_libexec_ayatana-indicator-datetime_ayatana-indicator-datetime-service.1000.crash.zip](https://github.com/AyatanaIndicators/ayatana-indicator-datetime/files/5144193/_usr_libexec_ayatana-indicator-datetime_ayatana-indicator-datetime-service.1000.crash.zip)

**Comment by saivinoba** [comment]
In journalctl I see error `Settings schema 'org.ayatana.indicator.datetime' is not installed`. 

@tari01, what does dropping 'show-clock' setting mean? That there will be no clock in the indicators or it will not be shown be default?

**Comment by tari01** [comment]
> In journalctl I see error `Settings schema 'org.ayatana.indicator.datetime' is not installed`.

Hmm... This smells like an installation issue. Could this be a duplicate of #15?

> @tari01, what does dropping 'show-clock' setting mean? That there will be no clock in the indicators or it will not be shown by default?

: ) Quite the opposite: there will always be a clock - that's the minimum we need to show.

**Comment by saivinoba** [comment]
@tari01, thank you for the clarification.

I don't have same error message as in your linked issue. But I see the following:
```
ಸೆಪ್ಟೆಂ 03 13:24:27 sai-HP-Laptop-14-bs0xx ayatana-indicat[1057]: Settings schema 'org.ayatana.indicator.datetime' is not installed
ಸೆಪ್ಟೆಂ 03 13:24:27 sai-HP-Laptop-14-bs0xx kernel: traps: ayatana-indicat[1057] trap int3 ip:7fde83fd10b5 sp:7ffcc1b56110 error:0 in libglib-2.0.so.0.64>
ಸೆಪ್ಟೆಂ 03 13:24:28 sai-HP-Laptop-14-bs0xx systemd[903]: ayatana-indicator-datetime.service: Main process exited, code=dumped, status=5/TRAP
░░ Subject: Unit process exited
░░ Defined-By: systemd
░░ Support: http://www.ubuntu.com/support
░░ 
░░ An ExecStart= process belonging to unit UNIT has exited.
░░ 
░░ The process' exit code is 'dumped' and its exit status is 5.
ಸೆಪ್ಟೆಂ 03 13:24:28 sai-HP-Laptop-14-bs0xx systemd[903]: ayatana-indicator-datetime.service: Failed with result 'core-dump'.
░░ Subject: Unit failed
░░ Defined-By: systemd
░░ Support: http://www.ubuntu.com/support
░░ 
░░ The unit UNIT has entered the 'failed' state with result 'core-dump'.
```

**Comment by saivinoba** [comment]
I was able to fix panel clock issue by removing 'indicator-datetime' package. I saw a `com.canonical.indicator.datetime.gschema.xml` file under '/usr/share/glib-2.0/schemas/' and thought probably that is conflicting with `org.ayatana.indicator.datetime.gschema.xml` when `"/usr/lib/x86_64-linux-gnu/glib-2.0/glib-compile-schemas" /usr/share/glib-2.0/schemas` runs.

The clock is now visible in the panel and in journal I see, `ಸೆಪ್ಟೆಂ 05 09:00:46 sai-HP-Laptop-14-bs0xx systemd[932]: ayatana-indicator-datetime.service: Succeeded.`

**Comment by sunweaver** [comment]
Partially addressed by 24d102c12a1470726c1d5321a918825cd3d18527.
